docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx

```diff
@@ -0,0 +1,25 @@
+import CodeBlock from '@theme/CodeBlock';
+
+<span>There are several ways of configuring dlt to use <code>{props.file_type}</code> file format for normalization step and to store your data at the destination:</span>
+
+1. <span>You can set the <code>loader_file_format</code> argument to <code>{props.file_type}</code> in the run command:</span>
+
+<pre language="py">
+info = pipeline.run(some_source(), loader_file_format="{props.file_type}")
+</pre>
+
+
+2. You can set the `loader_file_format` in `config.toml` or `secrets.toml`:
+
+<pre language="toml">
+[normalize]{'\n'}
+loader_file_format="{props.file_type}"
+</pre>
+
+3. You can set the `loader_file_format` via ENV variable:
+
+<pre language="sh">
+export NORMALIZE__LOADER_FILE_FORMAT="{props.file_type}"
+</pre>
+
+4. You can set the file type directly in [the resource definition](../../general-usage/resource#pick-loader-file-format-for-a-particular-resource).
```

Comment:
is it possible to add the code-snippet from the resource page? like:
```
@dlt.resource(file_format="{props.file_type}")
def generate_rows(nr):
    for i in range(nr):
        yield {'id':i, 'example_string':'abc'}
```
so readers don't have to go to another page

Reply:
I'm not sure if our snippet checker would be happy with it

```
<pre language="py">
@dlt.resource(file_format="{props.file_type}"){'\n'}
def generate_rows(nr):{'\n'}
&nbsp;&nbsp;&nbsp;&nbsp;for i in range(nr):{'\n'}
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;yield &#123;'id':i, 'example_string':'abc'&#125;
</pre>
```